packages/@react-aria/utils/src/useId.ts

```diff
@@ -31,7 +31,9 @@ export function useId(defaultId?: string): string {
     nextId.current = val;
   }, []);
 
-  idsUpdaterMap.set(res, updateValue);
+  if (typeof window !== 'undefined') {
```

Comment:
Unfortunately Deno have window object at server context, maybe use `useIsSSR` or `canUseDom`?

Reply:
thanks for the reminder about Deno